### DISCUSSION TITLE
archivos binarios en tpi

### DISCUSSION THREAD

**Discussion by SantiR2212** [INGCOM-UNRN-P1/cursada-2023#235] [Preguntas y respuestas] [answered]
se pueden usar archivos binarios para el codigo de la agenda? para guardar los datos de un contacto directamente como estructura en el archivo

**Comment by martinvilu** [comment, marked as answer]
Sí, pero tené en cuenta que un archivo de texto lo podés revisar a simple vista, uno binario, no tanto.
